### PR TITLE
Standardize credits header

### LINKED ISSUE #1122: Standardize Credits Page Header
### Overview
We need to standardize the Credits page header so that it will be more consistent with the rest of the website, and because a link to this page will soon be in the footer.

### Action Items
UI Design
- [x] Review the standard header layout in the Style Guide & Design System file on Figma
- [x] Apply the standards to the Credits page header on Figma
- [x] Discuss with @daniellex0 how to guide the changes in CSS for devs

Front-End Dev
- [x] Implement the new standard design of the Credits page header based on the [Figma design](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1074%3A0) (in pink rectangle)
   - [x] Reference the [Sitemap Page](https://www.hackforla.org/sitemap) header code, but try to use both a reusable `.header-container` class and another page-specific class (contact @daniellex0 / daniellenedivi on slack for clarification)

### Resources/Instructions
[Credits page](https://www.hackforla.org/credits)
[Credits Page on Figma
](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1074%3A0) (final design in pink rectangle)

Screen Shot 2021-03-06 at 7 47 21 PM

### PR DESCRIPTION
**Pull request by sena-ji**
Fixes #1122

### CODE REVIEW

File: pages/credits.html [hunk 1]
```diff
@@ -4,15 +4,17 @@
...
-  <div class="credit-hero">
-      <div class="step-links mobile-image">
-          <img class="credit-image" src="/assets/images/credits/credits-hero.png">
+  <div class="credits-text-margin">
+    <h1>Credits</h1>
+    <p class="header-text">Thank you to all of the artists who have made our page look great. 
+      Check out all of the iconography and imagery we have used on our site.
+    </p>
+  </div>
+  <!-- <div class="credit-hero">
```
Comment (akibrhast) [inline pages/credits.html]: It seems line 13, and 14 were commented out. If there is no need for them can you please delete those lines from this PR

File: pages/credits.html [hunk 2]
```diff
@@ -4,15 +4,17 @@
...
+      <div class="step-links mobile-image"> -->
+          <img src="/assets/images/credits/credits-hero.png">
 
-      </div>
+      <!-- </div> -->
```
Comment (akibrhast) [inline pages/credits.html]: Please delete line 17 commented out empty div

File: _sass/components/_credit-items.scss [hunk 3]
```diff
@@ -12,19 +12,54 @@
 }
 
 .credits-header-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
   background: #fff;
```
Comment (akibrhast) [inline _sass/components/_credit-items.scss]: Please replace raw color with it's contextual variable as described in this file
https://github.com/hackforla/website/blob/gh-pages/_sass/variables/_colors.scss#L9

File: _sass/components/_credit-items.scss [hunk 4]
```diff
@@ -12,19 +12,54 @@
...
   font-weight: normal;
   text-align: center;
 
+  img {
+    max-width: 350px;
+  }
+
+  @media #{$bp-below-mobile} {
+    // padding: 30px 15px;
+    // margin-top: 23%;
```
Comment (akibrhast) [inline _sass/components/_credit-items.scss]: Please remove commented out css styles

**Review by akibrhast — CHANGES_REQUESTED** [review]
Please remove commented out css styles in the files that you changed as well as commented out html. Also could you please ensure that the raw color in the scss file that you changed are converted to variable names.

The issue also mentions

>  Reference the Sitemap Page header code, but try to use both a reusable `.header-container` class and another page-specific class (contact @daniellex0 / daniellenedivi on slack for clarification)

From what I can tell looking at the code, the `.header-container` was not used. Was there a particular reason for this?

Thank you!

**Comment by sena-ji** [comment]
> Please remove commented out css styles in the files that you changed as well as commented out html. Also could you please ensure that the raw color in the scss file that you changed are converted to variable names.
> 
> The issue also mentions
> 
> > Reference the Sitemap Page header code, but try to use both a reusable `.header-container` class and another page-specific class (contact @daniellex0 / daniellenedivi on slack for clarification)
> 
> From what I can tell looking at the code, the `.header-container` was not used. Was there a particular reason for this?
> 
> Thank you!

For the .header-container, it seemed like there was a regular header-container and multiple header-container--insert page name. The reusable header-container does not give the wanted style for the page. Should I just make the changes with the header-container class? Or would you like me to make another separate class in the format: header-container--insert page name?

**Comment by akibrhast** [comment]
@sena-ji 
I believe the standardization we are trying to move towards is to use two classes that is you use the .header-container  and you also use another class header-container--page-name . The first is to apply a generic style to it that should be consistent across all pages. The second is to apply the page specific styles to it as needed.

I would hold on before implementing that change tough. Because I would like to confirm this with daniel and the rest of the team first


@daniellex0 @jbubar Please Advise.

<sub>** Note to Discuss [4/20/2021(Dev Meeting)] : https://github.com/hackforla/website/issues/1122#issuecomment-823564927</sub>

**Comment by daniellex0** [comment]
Yup exactly Akib, that's correct 👍 . But yeah Sena I mentioned to reach out to me in the comment because I knew it might be confusing, and we are still working on how to document all of these reusable classes for devs (it's causing a lot of confusion everywhere, you're not alone! But hopefully soon we'll have some useful/clear guidelines..)

**Review by akibrhast — APPROVED** [review]
Nice Work Here!

**Pull request merged**